pom.xml

```diff
@@ -262,7 +262,7 @@
                     <artifactId>maven-surefire-plugin</artifactId>
                     <version>2.22.1</version>
                     <configuration>
-                        <argLine>-Xmx512m -XX:MaxPermSize=512m -Djdk.net.URLClassPath.disableClassPathURLCheck=true</argLine>
```

Comment:
Was this change also needed by the tests to pass?

Reply:
I think we can keep this. Should I revert this?